README.md

```diff
@@ -21,6 +21,13 @@ RNAget maintains an up-to-date report of the compliance status of its reference
 Please review the [Compliance Documentation](https://rnaget-compliance-suite.readthedocs.io/en/latest/) for 
 installation and usage instructions.
 
+### Installation
+
+```
+pip install -r requirements_doc.txt
```

Comment:
@alextsaihi I think we can probably revert this back now that `install_requires` is updated

Reply:
HI @alextsaihi. Yes, I think that now `python setup.py install` should just work.

Reply:
Addressed in 1f41339